### DISCUSSION TITLE
[Feature] Conditional Albums

### DISCUSSION THREAD

**Discussion by athornfam2** [immich-app/immich#4564] [Feature Request]
### The feature

A feature I enjoyed from Synology Photos was conditional based albums. It gave you the ability, when creating an album, to filter on file type, date range, and ML/Tagging conditions (like living room, new york city, etc).

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1673
